global/schedules2slack/Chart.lock

```diff
@@ -0,0 +1,6 @@
+dependencies:
+- name: owner-info
+  repository: https://charts.eu-de-2.cloud.sap
+  version: 0.9.3
```

Comment:
This version does not exists: https://github.com/sapcc/helm-charts/blob/master/common/owner-info/Chart.yaml
```suggestion
  version: 0.2.3
```